### ISSUE TITLE
Unexpected HTTP status code: 500

### ISSUE THREAD

**Issue by Colorstorm** [galaxyproject/galaxy#9254]
Hi, 
I am try to export a galaxy history in python.
From some histories I get `bioblend.ConnectionError: Unexpected HTTP status code: 500:`
What does these error mean and is it possible to avoid the error?

Here is my code:
```
from bioblend.galaxy import GalaxyInstance

ip = '***.***.***.***'
api_key = '*************************'
history_id = '***************'
gi = GalaxyInstance(url=ip, key=api_key)
histories_list = gi.histories.get_histories(deleted=False)
jeha_id = gi.histories.export_history(history_id, gzip=True, include_hidden=False,   include_deleted=False, wait=True, maxwait=None)
f = open("test/" + history_id + '.tar.gz', 'wb')
gi.histories.download_history(history_id=history['id'], jeha_id=jeha_id, outf=f)
f.close()
```

Here is the full error message:
```
Traceback (most recent call last):
  File "/Path/to/python/file/exportTest.py", line 8, in <module>
    jeha_id = gi.histories.export_history(history_id, gzip=True, include_hidden=False, include_deleted=False, wait=True, maxwait=None)
  File "/Path/to/python/interpreter/venv/lib/python3.5/site-packages/bioblend/galaxy/histories/__init__.py", line 617, in export_history
    r = self._put(payload={}, url=url, params=params)
  File "/Path/to/python/interpreter/venv/lib/python3.5/site-packages/bioblend/galaxy/client.py", line 165, in _put
    return self.gi.make_put_request(url, payload=payload, params=params)
  File "/Path/to/python/interpreter/venv/lib/python3.5/site-packages/bioblend/galaxyclient.py", line 214, in make_put_request
    body=r.text, status_code=r.status_code)
bioblend.ConnectionError: Unexpected HTTP status code: 500: {"err_msg": "Uncaught exception in exposed API method:", "err_code": 0}
```

**Comment by nsoranzo** [comment]
@Colorstorm Yes, thanks. In fact it looks like it's a Galaxy bug, so I've transferred the issue to its repository. What version of Galaxy are you using?

**Comment by mvdbeek** [comment]
This issue has been fixed in https://github.com/galaxyproject/galaxy/pull/7367, and the first release with this fix was release 19.05. Feel free to re-open the issue if you're using 19.05 or newer and you still see this.
